README.md

```diff
@@ -22,8 +22,9 @@ https://github.com/sxyazi/yazi/assets/17523360/92ff23fa-0cd5-4f04-b387-894c12265
 
 ## Documentation
 
-- Usage: https://yazi-rs.github.io/docs/installation
+- Installation: https://yazi-rs.github.io/docs/installation
```

Comment:
This is indeed a "usage" document, not an "installation" document. It just so happens that the first section is an installation guide, but I believe it doesn't conflict; installation is simply the initial step to start using it.